### PR TITLE
feat: create fulfillment provider app docs

### PR DESCRIPTION
**Pull request by digitalpacman**

### CODE REVIEW

File: fulfillment-provider/open-api.json [hunk 1]
```diff
@@ -0,0 +1,3002 @@
+{
+  "openapi": "3.0.1",
+  "info": {
+    "title": "Order API",
+    "description": "This API specification describes the canonical ShipEngine eCommerce API Order Source module. A module which implements this specification can be integrated with the ShipEngine eCommerce Platform as an Order Source.",
```
Comment (joshuaflanagan) [inline fulfillment-provider/open-api.json:5]: This text seems wrong - it is referring to Order Soruces.

Comment (joshuaflanagan) [reply fulfillment-provider/open-api.json:5]: In fact, this entire API spec seems to be describing order sources

**Review by joshuaflanagan — APPROVED** [review]
The [API reference](https://api.redocly.com/registry/bundle/auctane-uff/Fulfillment%20Provider/V1/openapi.yaml?branch=master) has a typo. It says `describes the canonical ShipEngine Fulfillmen Provider module` - there should be a `t` in `Fulfillment`.

File: fulfillment-provider/open-api.json (hunk 1 above)
Comment (digitalpacman) [reply fulfillment-provider/open-api.json:5]: @joshuaflanagan this is a copy paste error. this file isnt actually used by anything. i dont even know why it exists in order source.

**Pull request merged**